### PR TITLE
Fix Incorrect error message for invalid QR code 

### LINKED ISSUE #7571: Unable to Scan The QR Code in Asset Module
Unable to Scan the QR Codes in the assets Module in Windows 11,Mac,iOS irrespective of any Browsers used. error Messages are popped as soon as the QR Code is scanned and detected. The Snapshot of the Error Message is pasted below

Screenshot 2024-04-08 173142

### PR DESCRIPTION
**Pull request by AnkurPrabhu**
## Proposed Changes

- Fixes #7571 
- not calling the backend api when the QR code is invalid 
-  stop infinite loading screen when the QR code  is invalid 

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [x] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

SS:
Screenshot 2024-04-12 at 1 36 44 AM

### CODE REVIEW

File: src/Components/Assets/AssetsList.tsx [hunk 1]
```diff
@@ -119,8 +119,11 @@ const AssetsList = () => {
         return data?.results[0].id;
       }
     } catch (err) {
-      console.log(err);
+      console.error(err);
+      Notification.Error({ msg: err });
     }
+    setIsLoading(false);
+    Notification.Error({ msg: "Invalid Asset Id" });
```
Comment (bodhish) [inline src/Components/Assets/AssetsList.tsx]: Lets make the message more friendly; 

Example 
> Oops! The asset ID you entered does not appear to be valid.

Also lets keep all strings in I18n

**Review by bodhish — CHANGES_REQUESTED** [review]

**Comment by AnkurPrabhu** [comment]
@bodhish i have updated the pr

**Review by rithviknishad — CHANGES_REQUESTED** [review]
strings not present in i18n files.

**Comment by rithviknishad** [comment]
Refer: https://github.com/coronasafe/care_fe/issues/4467

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
@AnkurPrabhu clear the merge conflict

CC: @rithviknishad

**Comment by AnkurPrabhu** [comment]
@nihal467 i have resolved the conflict

**Comment by nihal467** [comment]
@AnkurPrabhu remove the additional "something went wrong...!" error message , when we scan invalid QR

image

**Comment by AnkurPrabhu** [comment]
@rithviknishad can you tell me if my approach is correct ?

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM @khavinshankar can you resolve the merge conflict and merge it

**Review by khavinshankar — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Pull request merged**